pkg/models/model_gallery.go

```diff
@@ -111,6 +111,42 @@ func (g *Gallery) listZipContents() ([]*zip.File, *zip.ReadCloser, error) {
 		b := filteredFiles[j]
 		return utils.NaturalCompare(a.Name, b.Name)
 	})
+	cover := Contains(filteredFiles, "cover.jpg") // first image with cover.jpg in the name
+	if cover >= 0 {                               // will be moved to the start
+		reorderedFiles := Reorder(filteredFiles, cover)
+		if reorderedFiles != nil {
+			return reorderedFiles, readCloser, nil
+		}
+	}
 
 	return filteredFiles, readCloser, nil
 }
+
+func Contains(a []*zip.File, x string) int { // return index of first occurenece of string x in name of zip contents
```

Comment:
Does this need to be public (capitalised)?

Also, documentation for the function belongs on the line above.

Reply:
wasn't sure if it will useful in the future. fixing